Islandora/islandora_scholar
# Islandora 2338

**JIRA Ticket Islandora/ISLANDORA-2338**: (https://jira.duraspace.org/browse/ISLANDORA-2338)
Update screenshots in README.md and wiki page for RC 7.x-1.12 Islandora_scholar module

* Other Relevant Links 
Click here do download the OVA file for Islandora 7.x-1.12 RC to create the test environment.
https://drive.google.com/file/d/1zPqO4rfiZ5k61HUcYIj4QTtHqMh1fW1y/view

# Purpose
This is meant to keep the README.md file up to date.

# What's new?
REMOVE CURRENT IMAGE
![current_xpath_config](https://user-images.githubusercontent.com/11573234/48638772-95607a80-e99f-11e8-90b3-ad6e037524f0.JPG)

INSERT PROPOSED IMAGE
![proposed_xpath_config](https://user-images.githubusercontent.com/11573234/48638782-9beef200-e99f-11e8-959f-e8466cf1f7d4.JPG)


# How should this be tested?
1.  Bring up Islandora Release Candidate 7.x-1.12 in vagrant using the RC 7.x-1.12 OVA file.

2.  Navigate to HOME >> Administration >> Islandora >> Solution Pack Configuration >> Scholar
    (http://localhost:8000/node#overlay=admin/islandora/solution_pack_config/scholar)
    
3.  Move from the BASIC CONFIGURATION tab to the XPATH CONFIGURATION tab.
    (http://localhost:8000/node#overlay=admin/islandora/solution_pack_config/scholar/xpaths)

4.  Check the box labeled "Enable Custom Paths"

5.  Confirm that the proposed image shown in this pull request matches the content seen under the XPATH CONFIGURATION tab.

If these images match, the test is passed.

# Additional Notes:
This change will require an update to the wiki page 
https://wiki.duraspace.org/display/ISLANDORA/Islandora+Scholar
which is also addressed in  https://jira.duraspace.org/browse/ISLANDORA-2338

This change does not impact existing code in Islandora RC 7.x-1.12.

# Interested parties
@Islandora/7-x-1-x-committers
@Don Richards
@bryjbrown
